app/controllers/admin/announcements_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Admin::AnnouncementsController < Admin::Controller
+  load_and_authorize_resource :announcement, through: :current_tenant, class:
+      Instance::Announcement.name
+  add_breadcrumb :index, :admin_announcements_path
+
+  def index
+    @announcements = @announcements.includes(:creator)
+  end
+
+  def new
+  end
+
+  def create
+    if @announcement.save
+      redirect_to admin_announcements_path,
+                  notice: t('.notice', title: @announcement.title)
+    else
+      render 'new'
+    end
+  end
+
+  def edit
+  end
+
+  def update
+    if @announcement.update_attributes(announcement_params)
+      redirect_to admin_announcements_path,
+                  notice: t('.notice', title: @announcement.title)
+    else
+      render 'edit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.